### DISCUSSION TITLE
Default Theme Variables

### DISCUSSION THREAD

**Discussion by creazy231** [heroui-inc/heroui#214] [Q&A] [answered]
Hey @everyone 🙌 

I'd like to use NextUI with a combination of TailwindCSS Classes and therefore I need to add the default theme options like colors etc. to the tailwind.config.js.
Anyone know where NextUI stores its default values for the default light and dark theme to use them to generate TailwindCSS classes?

Thanks ❤️

**Comment by tianenpang** [comment, marked as answer]
Hey @creazy231 

The themes source code for NextUI is at `packages/react/src/theme` [check it out](https://github.com/nextui-org/nextui/tree/main/packages/react/src/theme), it should be what you are looking for.
